parser/parser.y

```diff
@@ -4111,6 +4112,15 @@ LowPriorityOptional:
 		$$ = true
 	}
 
+HighPriorityOptional:
```

Comment:
Because the grammar recognize only some kind of the the priority
For example, https://dev.mysql.com/doc/refman/5.7/en/select.html , 
Select statement just have HighPriority or NoPriority in the grammar, no LowPriority.

If you insist, I can simplify the grammar a bit.